jlm/llvm/Makefile.sub

```diff
@@ -61,18 +61,126 @@ LIBLLVM_SRC = \
+	jlm/llvm/ir/ssa.hpp \
+	jlm/llvm/ir/variable.hpp \
+	jlm/llvm/ir/basic-block.hpp \
+	jlm/llvm/ir/print.hpp \
+	jlm/llvm/ir/types.hpp \
+	jlm/llvm/ir/cfg-structure.hpp \
+	jlm/llvm/ir/operators/operators.hpp \
+	jlm/llvm/ir/operators/Phi.hpp \
+	jlm/llvm/ir/operators/load.hpp \
+	jlm/llvm/ir/operators/gamma.hpp \
+	jlm/llvm/ir/operators/GetElementPtr.hpp \
+	jlm/llvm/ir/operators/theta.hpp \
+	jlm/llvm/ir/operators/delta.hpp \
+	jlm/llvm/ir/operators/store.hpp \
+	jlm/llvm/ir/operators/alloca.hpp \
+	jlm/llvm/ir/operators/call.hpp \
+	jlm/llvm/ir/operators/sext.hpp \
+	jlm/llvm/ir/operators/lambda.hpp \
+	jlm/llvm/ir/aggregation.hpp \
+	jlm/llvm/backend/rvsdg2jlm/rvsdg2jlm.hpp \
+	jlm/llvm/backend/rvsdg2jlm/context.hpp \
+	jlm/llvm/backend/jlm2llvm/jlm2llvm.hpp \
+	jlm/llvm/backend/jlm2llvm/type.hpp \
+	jlm/llvm/backend/jlm2llvm/instruction.hpp \
+	jlm/llvm/backend/jlm2llvm/context.hpp \
 
-.PHONY: libllvm-release
-libllvm-release: CXXFLAGS += -O3
-libllvm-release: $(JLM_BUILD)/libllvm.a
+libllvm_TESTS += \
+	tests/jlm/llvm/backend/llvm/r2j/test-empty-gamma \
+	tests/jlm/llvm/backend/llvm/r2j/test-partial-gamma \
+	tests/jlm/llvm/backend/llvm/r2j/test-recursive-data \
+	tests/jlm/llvm/backend/llvm/jlm-llvm/TestAttributeConversion \
+	tests/jlm/llvm/backend/llvm/jlm-llvm/test-bitconstant \
+	tests/jlm/llvm/backend/llvm/jlm-llvm/test-function-calls \
+	tests/jlm/llvm/backend/llvm/jlm-llvm/test-select-with-state \
+	tests/jlm/llvm/backend/llvm/jlm-llvm/test-type-conversion \
+	tests/jlm/llvm/frontend/llvm/TestAttributeConversion \
+	tests/jlm/llvm/frontend/llvm/test-endless-loop \
+	tests/jlm/llvm/frontend/llvm/test-export \
+	tests/jlm/llvm/frontend/llvm/TestFNeg \
+	tests/jlm/llvm/frontend/llvm/test-function-call \
+	tests/jlm/llvm/frontend/llvm/test-recursive-data \
+	tests/jlm/llvm/frontend/llvm/test-restructuring \
+	tests/jlm/llvm/frontend/llvm/test-select \
+	tests/jlm/llvm/ir/operators/TestCall \
+	tests/jlm/llvm/ir/operators/test-ConstantFP \
+	tests/jlm/llvm/ir/operators/test-delta \
+	tests/jlm/llvm/ir/operators/TestGetElementPtr \
+	tests/jlm/llvm/ir/operators/TestLambda \
+	tests/jlm/llvm/ir/operators/test-load \
+	tests/jlm/llvm/ir/operators/TestPhi \
+	tests/jlm/llvm/ir/operators/test-sext \
+	tests/jlm/llvm/ir/operators/test-store \
+	tests/jlm/llvm/ir/test-aggregation \
+	tests/jlm/llvm/ir/test-cfg \
+	tests/jlm/llvm/ir/test-cfg-node \
+	tests/jlm/llvm/ir/test-cfg-orderings \
+	tests/jlm/llvm/ir/test-cfg-prune \
+	tests/jlm/llvm/ir/test-cfg-purge \
+	tests/jlm/llvm/ir/test-cfg-structure \
+	tests/jlm/llvm/ir/test-cfg-validity \
+	tests/jlm/llvm/ir/test-domtree \
+	tests/jlm/llvm/ir/test-ssa-destruction \
+	tests/jlm/llvm/ir/TestAnnotation \
+	tests/jlm/llvm/opt/alias-analyses/TestAgnosticMemoryNodeProvider \
+	tests/jlm/llvm/opt/alias-analyses/TestMemoryStateEncoder \
+	tests/jlm/llvm/opt/alias-analyses/TestRegionAwareMemoryNodeProvider \
+	tests/jlm/llvm/opt/alias-analyses/TestSteensgaard \
+	tests/jlm/llvm/opt/test-cne \
```

Comment:
There are tests missing. The `TestAndersen`, `TestPointerObjectSet`, and `TestPointsToGraph` tests are missing. I suspect that the list of tests is still the initial list from when you started this PR. You did not add the additional tests that were added since then to the tests list when updating the branch. Thus, there might be other tests missing as well.

Reply:
fixed this as well. The problem is that I forgot to delete the Makefile.sub to which these were added, so I did not get a merge conflict that would highlighted addition of the tests. Fixed now, also cleaned up orphaned makefiles so any other addition is going to cause a merge conflict in case I have to rebase again.